validator/client/src/main/java/tech/pegasys/teku/validator/client/BeaconProposerPreparer.java

```diff
@@ -229,6 +276,17 @@ private Optional<Eth1Address> getFeeRecipientFromProposerConfig(
     return config.getConfigForPubKey(publicKey).flatMap(Config::getFeeRecipient);
   }
 
+  private Optional<UInt64> getGasLimitFromProposerConfig(
+      final ProposerConfig config, final BLSPublicKey publicKey) {
+    return config
```

Comment:
i think this should likely just be the same as above where we just return whatever is in config, and the providing of default value happens if this is empty... it looks like the get is already returning default, so im not sure why we'd provide default here...